<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LcdConfig</class>
 <widget class="QDialog" name="LcdConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>显示屏模拟数据</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTableView" name="tableView">
       <property name="textElideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>设备名字：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>像素深度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="perlbits_cbb">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>32</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>24</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>16</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>设备宽：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="resxEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>设备高：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="resyEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="selectBt">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteBt">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addBt">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateBt">
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quitBt">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
